extensions/sql-database-projects/src/dialogs/updateProjectFromDatabaseDialog.ts

```diff
@@ -462,12 +468,14 @@ export class UpdateProjectFromDatabaseDialog {
 		this.compareActionRadioButton = view.modelBuilder.radioButton().withProps({
 			name: 'action',
 			label: constants.compareActionRadioButtonLabel,
+			ariaLabel: constants.compareActionRadioButtonLabel,
```

Comment:
removed them since they aren't needed. I was just adding ariaLabel on everything when I was going through this file and didn't realize this gets set automatically